src/python/WMCore/ReqMgr/Service/Auxiliary.py

```diff
@@ -79,6 +80,24 @@ def get(self):
         # "return result" only would return dict with keys without (!!) values set
         return rows([result])
 
+class ProcessMatrix(RESTEntity):
+    """
+    Process and thread matrix in reqmgr2
+    """
+    def __init__(self, app, api, config, mount):
+        # CouchDB auxiliary database name
+        RESTEntity.__init__(self, app, api, config, mount)
+        self.time0 = time.time()
+
+    def validate(self, apiobj, method, api, param, safe):
+        pass
+
+    @restcall(formats=[('text/plain', PrettyJSONFormat()), ('application/json', JSONFormat())])
+    def get(self):
+        sdict = {'server': processStatus()}
+        sdict['server'].update({'uptime': time.time() - self.time0})
+        sdict.update(threadStack())
```

Comment:
Seangchan, may be we need to provide threadStack only if we'll pass verbose/detail argument. It is good to have for debugging but in fact it is useless for metrics visualization. It is enough to report number of threads running I think.